### PR TITLE
Update fr_FR.yml

### PR DESCRIPTION
**Pull request by Niasio**

### CODE REVIEW

File: src/main/resources/lang/fr_FR.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
-# Translation by Amaury Carrade
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:1]: Why is my name removed from the head comment?

Comment (Niasio) [reply src/main/resources/lang/fr_FR.yml:1]: I forgot you lol

**Comment by RypoFalem** [comment]
Honestly, I don't know how to properly resolve the crediting conflict. I don't have any experience with this sort of thing. It's important to me is that everyone feels like they are getting the credit they deserve. Would it please both of you if the top 2 lines simply read:
```
# Original translation by AmauryCarrade
# Updated by Niasio -> https://github.com/Niasio
```

**Comment by Niasio** [comment]
Yes, it's a solution that seems very fair :D

**Comment by AmauryCarrade** [comment]
That's fine for me too :)

Or

```yaml
# Translations by Amaury Carrade (https://github.com/AmauryCarrade) and Niasio (https://github.com/Niasio)
```

File: src/main/resources/lang/fr_FR.yml [hunk 2]
```diff
@@ -55,6 +57,26 @@ target:
...
+  
+      2. Cliquez à gauche ou à droite à partir d'un support d'armure pour ouvrir le menu.
+      
+      3. Sélectionnez les options de menu marquées.
+      
+      4. Cliquez avec le bouton droit de la souris sur l'armure avec l'outil d'édition en main pour appliquer ces options.
+      
+      5. Cliquez avec le bouton droit de la souris pour modifier à l'envers!"
+helptips:
+  msg: "Astuces:
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:71]: There is a space missing here before `:` (required in French typography).

File: src/main/resources/lang/fr_FR.yml [hunk 3]
```diff
@@ -55,6 +57,26 @@ target:
...
+  
+  1. Appuyez sur le bouton \"Changer d'Item dans la Main\" (par défaut F) tout en tenant l'outil d'édition pour cibler un support d'armure spécifique, si d'autres étages d'armure sont dans le chemin.
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:73]: You should use french quotation marks. As example (you can copy paste if the character is missing from your keyboard layout):

```diff
- 1. Appuyez sur le bouton \"Changer d'Item dans la Main\" (par défaut F) tout en tenant l'outil d'édition pour cibler un support d'armure spécifique, si d'autres étages d'armure sont dans le chemin.
+ 1. Appuyez sur le bouton « changer un objet de main » (par défaut F) tout en tenant l'outil d'édition pour cibler un support d'armure spécifique, si d'autres armure sont sur le chemin.
```
In the meantime, I updated the key name to the one Minecraft uses in its French translation and improved the whole sentence a little bit.

File: src/main/resources/lang/fr_FR.yml [hunk 4]
```diff
@@ -55,6 +57,26 @@ target:
...
+help:
+  msg: "1. Tenez l'outil d'édition (<x>) dans votre main principale
+  
+      2. Cliquez à gauche ou à droite à partir d'un support d'armure pour ouvrir le menu.
+      
+      3. Sélectionnez les options de menu marquées.
+      
+      4. Cliquez avec le bouton droit de la souris sur l'armure avec l'outil d'édition en main pour appliquer ces options.
+      
+      5. Cliquez avec le bouton droit de la souris pour modifier à l'envers!"
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:69]: There is a space missing here before `!` (required in French typography).

File: src/main/resources/lang/fr_FR.yml [hunk 5]
```diff
@@ -55,6 +57,26 @@ target:
   msg: Valet cible verrouillé.
 notarget:
   msg: Valet cible déverrouillé.
+help:
+  msg: "1. Tenez l'outil d'édition (<x>) dans votre main principale
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:61]: There is a missing dot at the end of the sentence (and the dot is in others).

File: src/main/resources/lang/fr_FR.yml [hunk 6]
```diff
@@ -55,6 +57,26 @@ target:
...
+      
+      4. Cliquez avec le bouton droit de la souris sur l'armure avec l'outil d'édition en main pour appliquer ces options.
+      
+      5. Cliquez avec le bouton droit de la souris pour modifier à l'envers!"
+helptips:
+  msg: "Astuces:
+  
+  1. Appuyez sur le bouton \"Changer d'Item dans la Main\" (par défaut F) tout en tenant l'outil d'édition pour cibler un support d'armure spécifique, si d'autres étages d'armure sont dans le chemin.
+  
+  2. Vous pouvez mettre des étiquettes aux stands d'armure. Vous pouvez utiliser le symbole & pour créer des noms colorés.
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:75]: « stand d'armure » → « valets » ? Or « porte-armure », but I think the former is better.

File: src/main/resources/lang/fr_FR.yml [hunk 7]
```diff
@@ -55,6 +57,26 @@ target:
...
+  
+  3. Vous pouvez utiliser crouch + scroll wheel tout en maintenant l'outil d'édition pour changer Axis sans ouvrir le menu"
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:77]:
```diff
- 3. Vous pouvez utiliser crouch + scroll wheel tout en maintenant l'outil d'édition pour changer Axis sans ouvrir le menu"
+ 3. Vous pouvez vous accroupir puis utiliser la molette de la souris tout en maintenant l'outil d'édition pour changer d'axe de modification sans ouvrir le menu."
```
(Better and frenchier translation, plus the missing dot at the end.)

File: src/main/resources/lang/fr_FR.yml [hunk 8]
```diff
@@ -55,6 +57,26 @@ target:
...
+helpurl:
+  msg: "Plus d'info: https://github.com/RypoFalem/ArmorStandEditor/wiki"
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:79]: There is a space missing here before `:` (required in French typography).

Oh, and you should write « Plus d'infos » (plural form).

File: src/main/resources/lang/fr_FR.yml [hunk 9]
```diff
@@ -173,6 +195,10 @@ reset:
   msg: Réinitialisation de la pose
   description:
     msg: Réinitialise la position des membrs du corps aux valeurs par défaut
+helpgui:
+  msg: Aide!
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:199]: There is a space missing here before `!` (required in French typography). Also, is this `!` needed? I mean, it's there in the original English translation, but it feels a little bit weird.

May I suggest something like this, to keep the exclamation but with better phrasing.

```diff
- Aide!
+ À l'aide !
```

File: src/main/resources/lang/fr_FR.yml [hunk 10]
```diff
@@ -173,6 +195,10 @@ reset:
...
+  description:
+    msg: Cliquez pour avoir de une aide!
```
Comment (AmauryCarrade) [inline src/main/resources/lang/fr_FR.yml:201]: That's not a French sentence. I suggest:

```diff
- Cliquez pour avoir de une aide!
+ Cliquez ici pour obtenir de l'aide
```

**Review by AmauryCarrade — CHANGES_REQUESTED** [review]
While I was there, I left a few comments regarding the new French translations (as it's my native tongue).

Also, you use « armure » a lot to talk about armor stands, but they are armor… _stands_, so you may want to change this word to « valet » or « porte-armure ». :)

**Comment by RypoFalem** [comment]
Suggestions also implemented.

https://github.com/RypoFalem/ArmorStandEditor/commit/4be0f89b0b95ea88bda053217399a4c8a0bc58b2

**Comment by AmauryCarrade** [comment]
Thanks!

**Pull request merged**